kafkakewl-metrics/src/main/scala/com/mwam/kafkakewl/metrics/Main.scala

```diff
@@ -56,6 +56,7 @@ object Main extends ZIOAppDefault {
 
     (for
       topicInfoSource <- ZIO.service[KafkaTopicInfoSource]
+      _ <- HighOffsetMetricsExposer.startPublishing(topicInfoSource.subscribeStream())
```

Comment:
Do you know what happens to this fiber? It'll be interrupted automatically when the parent fiber finishes/is interrupted, but is this going to be OK, can the metrics service gracefully shutdown, interrupt fibers etc... correctly?

Reply:
This is now a scoped ZLayer in the same fashion as the `TopicInfoSourceCache`